File: py/reverseWords.py

def reverseWords(str):
    strL = str.split(' ')
    strLR = []
    strResult = ""
    for word in strL:
        strLR.append(word[::-1])
    for word in strLR:
        strResult += word + " "
    return strResult


result = reverseWords("let's leetcode")

File: js/reverseWords.js

function reverseWords(s) {
    let sList = s.split(' ');
    let wordA = [];
    let sListR = [];
    sList.forEach((word) => {
        wordA = word.split('');
        wordA.reverse();
        word = wordA.join('');
        sListR.push(word);
    });
    const result = sListR.join(' ');
    return result;
}

const result = reverseWords("Let's take LeetCode contest");